tailwind.config.cjs

```diff
@@ -21,7 +21,7 @@ const tailwindColors = require('tailwindcss/colors')
 module.exports = {
   content: [
     'packages/renderer/index.html',
-    'packages/renderer/src/**/*.{svelte,css}',
+    'packages/renderer/src/**/*.{svelte,ts,css}',
```

Comment:
hello, could you say why we added `ts` ?

Reply:
I really don't know the reasoning behind it but it's like we need to tell tailwind where we use its classes even if we just generate html strings.

By looking at the documentation (https://tailwindcss.com/docs/configuration#content) they clearly state "any file that contain Tailwind classes" 

`The content section is where you configure the paths to all of your HTML templates, JS components, and any other files that contain Tailwind class names.`

And by testing it it was true because the style was not applied otherwise
![image](https://github.com/containers/podman-desktop/assets/49404737/ce2bf27f-791a-4576-9944-272297c06e44)

The fact is that i see this effect only on some classes so not sure what's the exact behavior. But by adding the ts file everything works fine.